gen/DSE.Open.Values.Generators/Model/ValueTypeSpec.cs

```diff
@@ -21,6 +21,8 @@ internal abstract class ValueTypeSpec
 
     public string ContainedValueTypeName { get; set; } = default!;
 
+    public bool EmitEnsureIntialised { get; internal set; }
```

Comment:
Should default to true?

Reply:
I don't think so. Really, it should be required. It's always set to the correct state when building the spec.